src/TxEth.cpp

```diff
@@ -0,0 +1,40 @@
+#include "TxRx.h"
+
+//#define SERVER_IP "192.168.0.105" // IP адрес дом
```

Comment:
You can redefine SERVER_IP during compilation using -D flag. Add 2 modes for home and work in Makefile. But better transfer IP's to local file for safety and read IP from it.